package.json

```diff
@@ -231,6 +231,10 @@
                         "resource-commands-on-files": {
                             "type": "boolean",
                             "description": "If true, show Kubernetes resource commands on file context menu for all YAML files"
+                        },
+                        "preferBuildah": {
```

Comment:
I am not sure this is the right design here - what if someone comes along with a PR to support another build tool, say `img`?  How would we manage the interaction between `preferBuildah` and a hypothetical `preferImg`?

Suggest instead a `preferredImageBuildTool` with an enum that restricts it to be `Docker` or `Buildah` (cf. the `kubectlVersioning` config setting).